spec/database_setup.rb

```diff
@@ -0,0 +1,7 @@
+require 'pg'
+def truncate
+  con = PG.connect(dbname: "chitter_test")
+
+  con.exec("TRUNCATE peeps")
+  con.exec("TRUNCATE users")
```

Comment:
To keep the code DRY, you could have rather written con.exec("TRUNCATE peeps, users").